packages/datagateway-dataview/src/page/pageContainer.component.test.tsx

```diff
@@ -399,35 +375,25 @@ describe('PageContainer - Tests', () => {
         },
       ],
     });
-    const wrapper = createWrapper();
+    renderComponent();
 
-    wrapper
-      .find('[aria-label="selection-alert-link"]')
-      .first()
-      .simulate('click');
+    await user.click(
+      await screen.findByRole('button', { name: 'selection-alert-link' })
+    );
 
     expect(history.location.pathname).toBe('/download');
   });
 
-  it('passes correct landing page entities to breadcrumbs', () => {
+  it('shows breadcrumb according to the current path', async () => {
     history.replace(paths.toggle.isisInvestigation);
-    let wrapper = createWrapper();
+    renderComponent();
 
-    expect(wrapper.find('PageBreadcrumbs').prop('landingPageEntities')).toEqual(
-      ['investigation', 'dataset']
-    );
-
-    history.replace(paths.studyHierarchy.toggle.isisInvestigation);
-    wrapper = createWrapper();
-
-    expect(wrapper.find('PageBreadcrumbs').prop('landingPageEntities')).toEqual(
-      ['study', 'investigation', 'dataset']
-    );
+    expect(await screen.findByText('breadcrumbs.home')).toBeInTheDocument();
   });
```

Comment:
I don't think this test is testing much here really as it is (unless we're changing this test to just check that the breadcrumbs render in pageContainer) - if we actually wanted to test this then we'd either have to mock breadcrumbs to just display the landing page entities so we can verify they're correct, or do something different (e.g. check that when we're in the study hierarchy that study breadcrumbs have links etc.)